app/database/item_store_test.go

```diff
@@ -290,3 +291,44 @@ func TestItemStore_GetAccessDetailsForIDs_HandlesDBErrors(t *testing.T) {
 
 	assert.NoError(t, dbMock.ExpectationsWereMet())
 }
+
```

Comment:
As briefly discussed previously, I am not sure these unit tests (especially on stores) are very valuable when I see their cost of writing (quite lengthy) and maintenance. For instance, the fact they do the expected request depend on the "hasRows" implementation which is not the point of the implementation here, and anyway should be spotted by the BDD tests of the service using it.  
I would be in favor (for the future, the current ones are ok! 🙂) of just checking that running it on a mock db just not crash (that the query gen was able to generate something) and that a fail on the query is just returned, while ensuring 100% test coverage. This could be very generic to every such function and such tests could be reduced to 4-5 lines in total. Of course, in term of testing, that's not be perfect, but as all these functions are tested by services, I thing that is very acceptable. 
I am mainly worried that somebody else taking over this code will not maintain them otherwise (or will just delete them to prevent losing 30min in tests for a 2min fix of code), these tests convincing himself that testing is just a loss of time.